style.css

```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap');
+
+
+body {
+  font-family: 'Poppins';
+}
+
+header > div {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+  }
+  
+  nav {
```

Comment:
This nav is indented. Do you want to make it nested in the header tag,  "header > nav", or do you want to declare a generic nav styling?

Reply:
Hi! Nav is meant to be indented. I tried to keep all superior bar element styles together